.github/workflows/publish-helm-charts.yaml

```diff
@@ -43,8 +43,8 @@ jobs:
           echo "Packaging Helm charts..."
           mkdir -p .csi-op-release-packages
 
-          sed -i "s/^version:.*/version: ${VERSION}/" "deploy/charts/ceph-csi-operator/Chart.yaml"
-          sed -i "s/^version:.*/version: ${VERSION}/" "deploy/charts/ceph-csi-drivers/Chart.yaml"
+          sed -i "s/^version:.*/version: ${VERSION}/; s/^appVersion:.*/appVersion: v${VERSION}/" "deploy/charts/ceph-csi-operator/Chart.yaml"
```

Comment:
Do we want to follow only helm config bumps version and operator changes (like an image) bumps apiVersion or we want less maintenance and have both these same?

Reply:
We don't release helm charts separately,  it done along with the operator release,  it's same for both always.